packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -687,41 +680,7 @@ Information about project "Runner":
       expect(buildPhaseScriptContents.contains('ARCHS=armv7'), isTrue);
     });
 
-    testUsingOsxContext('sets ARCHS=x86_64 when sim local engine is set', () async {
```

Comment:
I moved this one into a new `sim local engine` group so the `localArtifacts` could be shadowed outside the test to be overridden in the context.